openquake/mbt/notebooks/sources_shallow_fault/mfd_double_truncated_from_slip_rate_SRC.ipynb

```diff
@@ -120,10 +121,15 @@
     "assert len(source_ids_list) == 1\n",
     "src = model.sources[source_ids_list[0]]\n",
     "\n",
-    "if hasattr(src, 'mmin') and src.mmin > 0.0:\n",
-    "    m_low = src.mmin\n",
+    "if hasattr(src, 'mmin') and src.mmin >= 0.0:\n",
+    "    m_min = src.mmin\n",
     "else:\n",
-    "    m_low = default_mmin\n",
+    "    m_min = 4.0\n",
```

Comment:
I think that if we want to set a default value for m_min it should be extremely low, say -5